docs/cargo-llvm-cov-show-env.txt

```diff
@@ -8,6 +8,10 @@ OPTIONS:
         --export-prefix
             Prepend "export " to each line, so that the output is suitable to be sourced by bash
 
+        --export-pwsh-prefix
+            Unicode escape and double quote values + prepend "$env:", so that the output is suitable
+            to be used with Invoke-Expression in pwsh.
```

Comment:
`in pwsh` to `in PowerShell`

Reply:
Since it requires the newer edition, I'm inclined to stick with `pwsh`. That's the common disambiugation for this needs powershell core vs windows powershell. E.g. you launch with `pwsh` and in ado pipelines it's `pwsh` is always powershell core whereas the `powershell` task is powershell core on mac + linux but windows powershell on windows. Does that change your thinking?

Reply:
Oh I'm sorry I misread that. This is the doc part. Agreed, I'll explicitly call that out